### PR TITLE
revert to a generic arch for openblas given problems reporte on non-lxplus machines

### PR DESCRIPTION
**Pull request by davidlange6**

### CODE REVIEW

**Comment by davidlange6** [comment]
unfortunately brings a 2x slowdown in one benchmark I tried..

**Comment by davidlange6** [comment]
please test

**Comment by davidlt** [comment]
This should not work non-x86_64 architectures. Thus only set `TARGET=PRESCOTT` for `amd64/x86_64`.

**Comment by davidlange6** [comment]
ah - yes, will fix

**Comment by davidlt** [comment]
Also looking at `TargetList.txt` in the package, you probably can safely target `PENRYN` (that's about 10 years old CPU). `PRESCOTT`  is basically Pentium 4 (2000 - 2008), no one should have such old machines.

File: open_mpi.spec~ [hunk 1]
```diff
@@ -0,0 +1,14 @@
+### RPM external OpenBLAS 0.2.19
```
Comment (davidlt) [inline open_mpi.spec~]: This file shouldn't be here.

**Comment by davidlange6** [comment]
please test

**Comment by smuzaffar** [comment]
please test
For PR tests we had PYTHONDONTWRITEBYTECODE=1 and I think this was causing py-numpy build to fail  (https://github.com/cms-sw/cmsdist/pull/2930#issuecomment-291188564) . I have removed it and trying running the tests again.

**Comment by smuzaffar** [comment]
last test failed due to cvmfs cache corruption
```
[root@vocms0xxx ~]# cvmfs_config fsck -p    
CernVM-FS fsck on /var/lib/cvmfs/shared
Verifying: .Fix: /var/lib/cvmfs/shared/02/4d1b1edb773cc775790376f6ce07030f5188b5 is corrupted, moved to quarantaine folder
.Fix: /var/lib/cvmfs/shared/04/1c0814165f27d55e8cc6eb25b81a604839a914 is corrupted, moved to quarantaine folder
....
```

**Comment by smuzaffar** [comment]
please test

**Pull request merged**